tapcfg/server.go

```diff
@@ -326,8 +326,30 @@ func genServerConfig(cfg *Config, cfgLogger btclog.Logger,
 
 	multiNotifier := proof.NewMultiArchiveNotifier(assetStore, multiverse)
 
-	// TODO(ffranr): Replace the mock price oracle with a real one.
-	priceOracle := rfq.NewMockPriceOracle(3600, 5820600)
+	// Determine whether we should use the mock price oracle service or a
+	// real price oracle service.
+	var priceOracle rfq.PriceOracle
+
+	switch cfg.Experimental.Rfq.PriceOracleAddress {
+	case rfq.MockPriceOracleServiceAddress:
+		priceOracle = rfq.NewMockPriceOracle(
+			3600, cfg.Experimental.Rfq.MockOracleCentPerSat,
+		)
+
+	case "":
```

Comment:
This is just for the itest? Because I think if the user leaves the CLI flag empty they would get an error from the `Validate()` method?

Reply:
No, I don't think the `Validate` method will return an error if the price oracle address is unset. See https://github.com/lightninglabs/taproot-assets/pull/915#discussion_r1618686552

Reply:
Ah, right, forgot about the standalone `tapd` users that don't care about RFQ at all. Makes sense.